db/schema.rb

```diff
@@ -12,24 +12,21 @@
 
 ActiveRecord::Schema.define(version: 2019_10_25_070223) do
 
-  # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
-
-  create_table "locales", force: :cascade do |t|
+  create_table "locales", options: "ENGINE=InnoDB DEFAULT CHARSET=latin1", force: :cascade do |t|
```

Comment:
Can you please add this file to the .gitignore?

Reply:
Sure.

Reply:
Also, can you please squash the commits?

Reply:
Right, we missed doing squash yesterday.